### DISCUSSION TITLE
Hey I'm just new in this environment and i have trouble in  Zksync Plugin Lecture ahh where i just deploy our first ZKc  but i got and error

### DISCUSSION THREAD

**Discussion by audit-bykhizar** [Cyfrin/foundry-full-course-cu#2805] [General]
![image_2024-11-04_163000394](https://github.com/user-attachments/assets/d64b052d-5999-48f4-b599-77e1fc576433)

**Comment by EngrPips** [comment]
Hello @khixarcyber, You need to switch your Metamask network to `zksync sepolia` and then retry your deployment

**Reply by audit-bykhizar** [reply]
Thnaks bro its works

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.

**Reply by audit-bykhizar** [reply]
Hey bro, can you tell me if we’ll have to use any real money when we take this course? kike any paid tools or any software or maybe some real tokens to deploy any projects ??

**Reply by EngrPips** [reply]
You won't be spending any real money.

**Reply by audit-bykhizar** [reply]
Tell me about yourself—are you still learning, or have you already become a developer?

**Reply by EngrPips** [reply]
I am still learning just like you, I hope and pray I become a developer someday.

**Reply by audit-bykhizar** [reply]
Keep going I know you'll make it!  
btw where u from?

**Reply by EngrPips** [reply]
I pray I make it. I am from Nigeria. where are you from?

**Reply by audit-bykhizar** [reply]
I'm from Pakistan
nice u meet you bro

**Reply by EngrPips** [reply]
It's all my pleasure. I trust you will make it too.

**Reply by audit-bykhizar** [reply]
Hey bro how you doing i just got new problem there remix show me some error and not be to compile i think there is some version issue
![image_2024-11-07_131901356](https://github.com/user-attachments/assets/4f073a59-8a63-42b6-943d-fd5e62f270ee)

**Reply by EngrPips** [reply]
If you hover the error what did it say?

**Comment by audit-bykhizar** [comment]
![image_2024-11-07_132236891](https://github.com/user-attachments/assets/92d7f48e-07e1-49bd-9450-5b8371b66f2b)

**Reply by EngrPips** [reply]
Please show you folder like I do below.

<img width="1440" alt="Screenshot 2024-11-07 at 09 19 48" src="https://github.com/user-attachments/assets/fad953ae-aa52-4827-9942-482501646ae9">

**Reply by audit-bykhizar** [reply]
![Uploading image_2024-11-07_133803687.png…]()

**Reply by EngrPips** [reply]
This image is not loading for me.

**Reply by audit-bykhizar** [reply]
![image_2024-11-07_133828456](https://github.com/user-attachments/assets/1f768bdd-d833-41f3-a202-be5680eca26b)

**Reply by EngrPips** [reply]
Please work in the `default_workspace` and put your files inside the `contracts` folder just like I did here

> Please show you folder like I do below.
> 
> <img alt="Screenshot 2024-11-07 at 09 19 48" width="1440" src="https://private-user-images.githubusercontent.com/95760918/383872814-fad953ae-aa52-4827-9942-482501646ae9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5Njg3ODcsIm5iZiI6MTczMDk2ODQ4NywicGF0aCI6Ii85NTc2MDkxOC8zODM4NzI4MTQtZmFkOTUzYWUtYWE1Mi00ODI3LTk5NDItNDgyNTAxNjQ2YWU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA3VDA4MzQ0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQzZjlmNGE0MjFhNDM1N2NlZjQ2ZDk3ZDYyOTMxYmZjN2E0OTEwNmQ5MDZmNTE0NjFmOGRiMTBjNTY3NGY1NjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ugFM35dsyNYJJ0mApvTft6R8ANowOiEm8clK4ozDb-k">

**Reply by audit-bykhizar** [reply]
ohh thanks bro i will get back you btw thanks a lot man

**Reply by EngrPips** [reply]
Anytime, I am looking forward to hearing from you.

**Reply by audit-bykhizar** [reply]
I don't why they still showing error 
![image_2024-11-07_140402248](https://github.com/user-attachments/assets/7affc1b7-8903-448c-8462-fd9a23fdc485)

**Reply by audit-bykhizar** [reply]
😭

**Reply by EngrPips** [reply]
Sometimes the error are just Remix glitch and it will go but you haven't put your files inside the `contracts` folder

**Reply by EngrPips** [reply]
Why are you importing two files from `./SimpleStorage.sol`?

**Reply by audit-bykhizar** [reply]
ohh so what i have to do

**Reply by audit-bykhizar** [reply]
ohh i see

**Reply by audit-bykhizar** [reply]
thanks man god bless you

**Reply by EngrPips** [reply]
🤔

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.

**Reply by audit-bykhizar** [reply]
![image_2024-11-07_140935477](https://github.com/user-attachments/assets/03b2752c-5a69-454c-9ea8-7c0ddc2b9bca)
it has been compiled ❤️❤️

**Reply by EngrPips** [reply]
Awesome.

**Comment by audit-bykhizar** [comment]
Hy bro how you doing . I hope you are good  
Actually when I went to downland Foundry in My Windows according to the course video and i copy the code form foundry web and past to the terminal I got an error

![image_2024-11-18_134843964](https://github.com/user-attachments/assets/c4fa686b-1286-485c-91b1-5ab8ac08c662)

**Reply by EngrPips** [reply]
Hello @khixarcyber, Read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2518#discussioncomment-10713419 and let's see if it helps you.
